test/Interfaces.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.13;
+
+import { IERC20 } from "erc20-helpers/interfaces/IERC20.sol";
+
+interface IAToken is IERC20 {
```

Comment:
I would just import this from `aave-v3-core` lib. It's why we added to have the interfaces available.